### ISSUE TITLE
Lab 4 - Error With "Select Input" and "render plot" 

### ISSUE THREAD

**Issue by lindaalvarez** [Watts-College/paf-513-template#10]
Hello, 

  I am unfortunately having issues when trying to run my code. In the beginning, it states
 "Error: unexpected symbol in:
"SelectInput( input$my_widget_id <- "Chicago White Sox"  # user selected value
             inputId"

 after writing down the code : 
SelectInput( input$my_widget_id <- "Chicago White Sox"  # user selected value
             inputId="my_widget_id",
             label="My Shiny Widget",
             choices=c("New York Yankees","Boston Red Sox","Chicago White Sox"),
             selected="Chicago White Sox"   
           )
I am also getting errors when trying to render the plot for the already written code. It simply says

 "Error in renderPlot({ : could not find function "renderPlot"

I would greatly appreciate some help. Thank you very much 
 @JasonSills 

<img width="1235" alt="Screen Shot 2024-01-31 at 9 09 10 PM" src="https://github.com/Watts-College/paf-513-template/assets/149988718/353e0c1c-0f93-4a93-9cf7-eee694bd812c">

**Comment by JasonSills** [comment]
@lindaalvarez - have you installed Shiny and Flexdashboard?

**Comment by bbenall4** [comment]
@lindaalvarez 
Hello,
Just wondering if you figured this out?

**Comment by lindaalvarez** [comment]
@JasonSills yes, thank you so much for checking in. I went ahead and emailed you about other issues I unfortunately encountered but worked my way through the code as best I could
